src/test/libcephfs/test.cc

```diff
@@ -3017,6 +3017,17 @@ TEST(LibCephFS, Readlinkat) {
   target[target_len] = '\0';
   ASSERT_EQ(0, memcmp(target, rel_file_path, target_len));
 
+#if defined(__linux__) && defined(O_PATH)
+  // test readlinkat with empty pathname relative to O_PATH|O_NOFOLLOW fd
+  fd = ceph_open(cmount, link_path, O_PATH | O_NOFOLLOW, 0);
+  ASSERT_LE(0, fd);
```

Comment:
oh, this is a bit confusing - ASSERT_GT(fd, 0) would have been better, but this entire routine uses such, so... :/

Reply:
I followed the style used elsewhere in this `TEST(LibCephFS, Readlinkat)` test routine.

Reply:
nvm 👍